File: servico-musica-java/java.rest/src/main/java/servico/java/rest/controller/UsuarioController.java
package servico.java.rest.controller;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import servico.java.rest.entity.Usuario;
import servico.java.rest.repository.UsuarioRepository;

import java.util.List;

@RestController
@RequestMapping("/servico-rest-java/usuario")
public class UsuarioController {

    @Autowired
    private UsuarioRepository usuarioRepository;

    @GetMapping("/todos-usuarios")
    public List<Usuario> todosUsuarios(){
        return usuarioRepository.findAll();
    }
}

File: servico-musica-kotlin/kotlin.rest/src/main/kotlin/servico/kotlin/rest/controller/UsuarioController.kt
package servico.kotlin.rest.controller

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import servico.kotlin.rest.entity.Usuario
import servico.kotlin.rest.repository.UsuarioRepository

@RestController
@RequestMapping("/servico-rest-kotlin/usuario")
class UsuarioController {
    @Autowired
    private val usuarioRepository: UsuarioRepository? = null

    @GetMapping("/todos-usuarios")
    fun todosUsuarios(): List<Usuario> {
        return usuarioRepository!!.findAll()
    }
}
